test/test-image.sh

```diff
@@ -20,3 +20,12 @@ if [[ "${CONTAINER_VERSION}" != "${EXPECTED_VERSION}" ]] ; then
   exit 1
 fi
 
+CLI_HELP="$(docker run --rm ${IMAGE} --help | sed 's/[ \t]*$//')"
```

Comment:
This code should not be added without adding `./docs/command-line-flags.txt`

Reply:
I don't think it's even needed

Reply:
OK. I have updated.